trino-connector/src/main/java/org/apache/gravitino/trino/connector/GravitinoConfig.java

```diff
@@ -69,6 +69,15 @@ public class GravitinoConfig {
           "true",
           false);
 
+  private static final ConfigEntry GRAVITINO_CLUSTER =
+      new ConfigEntry(
+          "gravitino.cluster",
+          "The cluster name of the Trino cluster, if it's empty, all the queries belongs the catalogs will be handled by the "
+              + "current Trino cluster, and if the cluster name is not equal to the catalog's cluster, the queries in the catalog "
+              + "will be handle by a remote Trino cluster",
```

Comment:
If the user sets the cluster name as "my_test_trino," how does the connector determine the corresponding URI for "my_test_trino"?

Reply:
The connector will check the url when the connector create. make sure the url is valid.

Reply:
You're missing the point, my question is, if a user configures `gravitino.cluster`, how should the URL corresponding to this cluster be configured?

Reply:
Not sure. the configuration is just a cluster name.